src/Neo/Ledger/MemoryPool.cs

```diff
@@ -9,6 +9,7 @@
 // Redistribution and use in source and binary forms with or without
 // modifications are permitted.
 
+#nullable enable
```

Comment:
Should add `#nullable disable` at the bottom of this file. To set back to normal.

Reply:
This is not a stric requiremnt, this nullable only work in the file. When Rider suggest me to add this, they also only do it with enable, without adding extra disable, So i think this should be fine.